test/factories/locations.rb

```diff
@@ -7,5 +7,6 @@
     city { Faker::Address.city }
     state { Faker::Address.state }
     zip { Faker::Address.zip_code }
+    provides_housing { [true, false].sample }
```

Comment:
`Faker::Boolean.boolean` would have worked, too, but I find the simplicity of this refreshing. 👍